### PR TITLE
Issue #257: Clean up People menu items.

### PR DESCRIPTION
**Pull request by jenlampton**
Running tests after latest pull/merge

### CODE REVIEW

File: core/modules/user/tests/user.test [hunk 1]
```diff
@@ -1791,7 +1791,11 @@ class UserRoleAdminTestCase extends BackdropWebTestCase {
     // correctly distinguish between role names and IDs.)
     $role_name = '123';
     $edit = array('name' => $role_name);
+<<<<<<< HEAD
     $this->backdropPost('admin/people/permissions/roles', $edit, t('Add role'));
+=======
+    $this->drupalPost('admin/config/people/roles', $edit, t('Add role'));
+>>>>>>> Issue #257: Clean up People menu items.
```
Comment (quicksketch) [inline core/modules/user/tests/user.test]: Conflict here needs to be fixed. Is this test even running? Seems like it should have errored-out.

File: core/modules/user/tests/user.test [hunk 2]
```diff
@@ -1837,7 +1856,11 @@ class UserRoleAdminTestCase extends BackdropWebTestCase {
 
     // Change the role weight and submit the form.
     $edit = array('roles['. $rid .'][weight]' => $old_weight + 1);
+<<<<<<< HEAD
     $this->backdropPost('admin/people/permissions/roles', $edit, t('Save order'));
+=======
+    $this->drupalPost('admin/config/people/roles', $edit, t('Save order'));
+>>>>>>> Issue #257: Clean up People menu items.
```
Comment (quicksketch) [inline core/modules/user/tests/user.test]: Lots of conflicts in this file need to be updated.

File: core/modules/user/user.module [hunk 3]
```diff
@@ -1572,6 +1581,19 @@ function user_menu() {
...
 /**
+ * Implements hook_menu_alter().
+ */
+function user_menu_alter(&$items) {
+  if (module_exists('field_ui')) {
+    // Accounts (field UI).
+    $items['admin/config/people/manage'] = $items['admin/config/people/manage/fields'];
+    $items['admin/config/people/manage']['type'] = MENU_NORMAL_ITEM;
+    $items['admin/config/people/manage']['weight'] = -20;
+    $items['admin/config/people/manage']['description'] = 'Configure display and behavior of user accounts including configuration of <em>fields</em> like user pictures.';
```
Comment (quicksketch) [inline core/modules/user/user.module]: Shouldn't the changes to user_entity_info() fix this so that hook_menu_alter() is not necessary.

**Comment by jenlampton** [comment]
I'm not sure why those tests weren't failing.  I worked out the merge conflicts and got them passing locally.  Pushing back up to Travis CI. (also yes, I removed the hook_menu_alter and tests still all pass)

**Comment by jenlampton** [comment]
Tests passed, and it looks like all the user tests were run this time.

**Comment by quicksketch** [comment]
I tested this out locally and found that without the hook_menu_alter(), the Manage fields item is not shown when visiting admin/config in the User settings group in the top left. So looks like it's needed after all. I tried adding back in the hook_menu_alter(), but then unfortunately we ended up with multiple items for "Manage fields" in admin menu.

user-manage-fields-menu

Not quite sure what the solution is quite yet.

**Comment by quicksketch** [comment]
New PR at https://github.com/backdrop/backdrop/pull/363.

**Pull request closed without merging**